CHANGELOG.md

```diff
@@ -13,6 +13,13 @@ Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
 
+## [v4.13.0.5] - 2021-03-31
+### Changed
+- rebased with official coreboot repository commit e7a68ec
+
+### Added
+- [resource allocator v4 support](https://doc.coreboot.org/releases/coreboot-4.13-relnotes.html#resource-allocator-v4)
```

Comment:
It is not merged: https://github.com/pcengines/coreboot/pull/465
Please remove

Reply:
You right. Sorry, I was sure that this was merged. Removed.